### PR TITLE
Fix composer to delete require php extensions

### PR DESCRIPTION
**Pull request by goncafa**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,14 +3,7 @@
...
     "require": {
-        "php": ">=5.5.0",
-        "ext-curl": "*",
-        "ext-json": "*",
-        "ext-mbstring": "*",
-        "ext-soap": "*",
-        "ext-dom": "*",
-        "ext-mcrypt": "*",
-        "ext-openssl": "*"
+        "php": ">=5.5.0"
```
Comment (leosoto) [inline composer.json]: Why did you delete *all* of the extensions and not just the ones added for webpay? I think that `curl` and `json` and `mbstring` are required by onepay anyway

Comment (goncafa) [reply composer.json]: As I said, this only works to fail the install of composer library if in the host does not have those installed. For that reason i have removed all the extension here becouse if you have not the extension installed you could install lib anyway but will fail when you use it and it should say that XXXX function does not found.

**Comment by leosoto** [comment]
Yes but that’s a not standard way to do things. We can do such non standard thing for the webpay thing (it’s good to not disturb onepay users because of the webpay mess). But I’m not sure that removing it on all cases is a good idea.

Even more: when the webpay stuff is updated to php7.x compatible I’d keep the libs required by webpay on this file so that the installation fails when the exts are not present. That way we’ll learn if the exts we are using are not commonly installed (and only in that case go back to not declaring them in this file)

File: composer.json (hunk 1 above)
Comment (goncafa) [reply composer.json]: Ok, i have put the old extensions again. Its ready to review again.
